src/fhir/1_Questionnaire-C-IDAS.json

```
@@ -253,12 +253,12 @@
       "type": "choice",
       "required": false,
       "linkId": "cs-idas-4",
-      "text": "Irritability - easily annoyed, short-tempered",
+      "text": "In the last week have you observed <b>irritability</b> in your care recipient? (easily annoyed, short-tempered)",
```

same here no HTML for `text` field (whereas `valueString` under the `_text` node can)
```suggestion
      "text": "In the last week have you observed irritability in your care recipient? (easily annoyed, short-tempered)",
```